ios/browser/api/url_sanitizer/url_sanitizer_service.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_H_
+#define BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_H_
+
+#import <Foundation/Foundation.h>
+
+@interface URLSanitizerService : NSObject
```

Comment:
Missing `OBJC_EXPORT` on this interface?

Reply:
fixed